contrib/restic/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "restic"
+pkgver = "0.16.2"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "Backup solution written in Go"
+maintainer = "triallax <triallax@tutanota.com>"
+license = "BSD-2-Clause"
+url = "https://restic.net"
+source = f"https://github.com/restic/restic/releases/download/v{pkgver}/restic-{pkgver}.tar.gz"
+sha256 = "88165b5b89b6064df37a9964d660f40ac62db51d6536e459db9aaea6f2b2fc11"
+
+
+def do_build(self):
+    from cbuild.util import golang
+
+    self.golang.build(wrksrc="cmd/restic")
```

Comment:
you can probably use `make_build_args = ["./cmd/..."]` instead (so ./cmd/restic ?)